types/picomatch/lib/picomatch.d.ts

```diff
@@ -84,7 +84,10 @@ declare namespace picomatch {
          * The function receives the range values as two arguments, and it must return a string to be used in the generated regex.
          * It's recommended that returned strings be wrapped in parentheses.
          */
-        expandRange?: ((a: string, b: string) => string) | undefined;
+        expandRange?:
+            | ((from: string, to: string, options: PicomatchOptions) => string)
+            | ((from: string, to: string, step: string, options: PicomatchOptions) => string)
+            | undefined;
```

Comment:
I believe this could be done with function overloading. Would probably give a better ts experience.
See https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads